### PR TITLE
don't recommend the obsolete dotty homebrew formula anymore

### PR DESCRIPTION
**Pull request by SethTisue**
a Scala 2 user on Discord, looking to move to Scala 3, was bitten by this

### CODE REVIEW

**Review by bishabosha — APPROVED** [review]
We could also update the scala 2 download page to use scala@2.13 but this would need more logic otherwise it will be invalid for 2.12 🥲

**Comment by SethTisue** [comment]
Yeah :-(

File: _includes/downloads-scala3.html [hunk 1]
```diff
@@ -29,7 +29,7 @@ <h2>Ways to Install This Release</h2>
         <code>sdk install scala</code>
       </li>
       <li>On macOS you can also use <a href="https://brew.sh/">Homebrew</a> and run the following commands:<br>
-        <code>brew update</code><br /><code>brew install lampepfl/brew/dotty</code>
+        <code>brew update</code><br /><code>brew install scala</code>
```
Comment (julienrf) [inline _includes/downloads-scala3.html:32]: Where is this formula defined?

Comment (bishabosha) [reply _includes/downloads-scala3.html:32]: It's in homebrew/homebrew-core

**Pull request merged**